content/en/api/users/users_getall.md

```diff
@@ -12,22 +12,3 @@ Get the list of all users in the organization. This list includes all users even
 **ARGUMENTS**:
 
 This endpoint takes no JSON arguments.
```

Comment:
This endpoint actually takes several arguments for sorting / filtering / pagination of the results.

* page[size]
    * Number of records to return
* page[number]
    * Page offset for records
* sort
    * User attribute to order results by
* sort_dir
    * Direction of ordering
    - asc, desc
* filter
     * String to search for in records
* filter[status]
    * Filter on Status attribute. Comma separated list
    - Active,Pending,Disabled